cosmos/dbt/selector.py

```diff
@@ -210,14 +228,23 @@ def load_from_statement(self, statement: str) -> None:
         items = statement.split(",")
 
         for item in items:
-            if item.startswith(PATH_SELECTOR):
-                self._parse_path_selector(item)
-            elif item.startswith(TAG_SELECTOR):
-                self._parse_tag_selector(item)
-            elif item.startswith(CONFIG_SELECTOR):
-                self._parse_config_selector(item)
-            else:
-                self._parse_unknown_selector(item)
+            regex_match = re.search(GRAPH_SELECTOR_REGEX, item)
+            if regex_match:
+                precursors, node_name, descendants = regex_match.groups()
+                if node_name is None:
+                    ...
+                elif precursors or descendants:
+                    self._parse_unknown_selector(item)
+                elif node_name.startswith(PATH_SELECTOR):
+                    self._parse_path_selector(item)
+                elif "/" in node_name:
+                    self._parse_path_selector(f"{PATH_SELECTOR}{node_name}")
```

Comment:
Please, could you add a test covering this use-case?

Reply:
Thanks for the review!  I've added a test for this now here: https://github.com/astronomer/astronomer-cosmos/blob/67cb56ff9c4d7ab20dcd3da926a83aa83bea80b6/tests/dbt/test_selector.py#L194